docs/repo/debugging/async-test-hangs.md

```diff
@@ -0,0 +1,270 @@
+# Debugging Async Test Hangs
+
+(Adapted from an internal document originally written by @jaredpar)
+
+Tests hanging on CI machines are often caused by async tests that are blocked waiting for results that will never complete. Unlike traditional synchronous hangs, attaching to an async hang and looking at executing threads typically will not provide anything useful as there won't be any threads actually executing the "awaiting"  methods.
+
+This document walks through debugging a test hang using WinDBG and SOS.
+
+
+## Prerequisites 
+
+- [WinDBG](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/debugger-download-tools)
```

Comment:
Is this really the primary external link to get WinDBG?

Reply:
As far as I can see yes. This seems to be no other page like this on Microsoft.